STEP_Project/preferred.js

```diff
@@ -32,9 +32,32 @@ function createForm()
             let breakLine = document.createElement('br');
             selectForm.appendChild(breakLine);
         }
+
+        selectForm.addEventListener('click',enableApplyButton);
     });
 }
 
+/**
+ * Enables applyButton if one of the preferred entities in the form has been selected.
+ * @param {event} event - This is expected to be a click event.
+ */
+function enableApplyButton(event)
+{
+    let selectForm = document.getElementById('radioButtons');
+    let numButtons = selectForm.length;
+
+    for (let i = 0; i < numButtons; i++)
+    {
+        if (selectForm[i].checked == true)
+        {
+            let applyButton = document.getElementById("apply");
+            applyButton.disabled = false;
+            applyListener(applyButton); 
+            return;
+        }
+    }
+}
+
```

Comment:
Add tests for this and update screenshot of tests in PR description.

Reply:
Added testing and uploaded the new screenshot